Atomic/backends/_ostree.py

```diff
@@ -150,7 +172,8 @@ def get_layers(self, image):
 
     @staticmethod
     def get_dangling_images():
-        return []
+        syscontainers = SystemContainers()
```

Comment:
looking at it, it seems the function should not be static.  Could you please make it non static and use `self.syscontainers` as @yuqi-zhang suggested?